### ISSUE TITLE
User can't delete his own bookings

### ISSUE THREAD

**Issue by miotto** [Wunderbyte-GmbH/moodle-mod_booking#12]
After the changes in commit 4b0cb5743a910302331b4a37454b0f1398775790 , a user can not delete his own bookings.
The changes (3 times) were:

```
- if (($booking->allowupdate and $option->status != 'closed') or has_capability('mod/booking:deleteresponses', $context)) {
+ if (has_capability('mod/booking:deleteresponses', $context) and $booking->allowupdate and $option->status != 'closed') {
```

A user currently has no rights that the buttons are displayed.

**Comment by miotto** [comment]
Fixed with commit 20c5e03799ceb11ab210bdc64164a49bc6d746c3

**Comment by miotto** [comment]
Thanks
